### DISCUSSION TITLE
5기 1주차 회고

### DISCUSSION THREAD

**Discussion by imakerjun** [next-step/blog#216] [회고]
```
# 📅 4월 12~19일의 회고
## 📝 블로그 글 소개

## 💡 인사이트 있었던 일

## 🤔 아쉬움이 남은 일

## 🪜 다음주에 다르게 하고 싶은 것이 있다면?(액션 플랜과 함께)
```

**Comment by oen142** [comment]
# 📅 4월 12~19일의 회고
## 📝 블로그 글 소개
AWS 발표 준비로 인해 블로그는 버퍼 기간을 가졌습니다.
발표 후기를 남겨야겠네요

## 💡 인사이트 있었던 일
AWS CodeDeploy를 사용하면서 네트워크에 대해서 좀더 깊이 공부할수 있었다.
쉘 스크립트를 공부 시작 했는데 재밌긴 하다.
한솔님이랑 개발자 진로(이직)에 대해서 얘기를 했는데 내가 하고싶은게 너무 많다. 
(백엔드 개발 , AWS 클라우드 , SE 등등..  ) 다 한다고 좋은 개발자일까(?)
하나만 잘해도 버겁던데

## 🤔 아쉬움이 남은 일
발표를 준비하는데 PPT를 오랜만에 작성하느라 자신이 없다. (말로 잘 풀어야지)
디자인 잘하는 사람이 부럽다.

## 🪜 다음주에 다르게 하고 싶은 것이 있다면?(액션 플랜과 함께)
AWS Lambda 공부후 배포까지 해보자.

**Comment by loop-study** [comment]
# 📅 4월 12~19일의 회고
## 📝 블로그 글 소개
이규원의 현실 세상의 TDD, 기초편을 정리했습니다.

## 💡 인사이트 있었던 일
기초 개념부터 학습예제로 쉽게 TDD는 무엇인지 알 수 있었습니다.

## 🤔 아쉬움이 남은 일
심화편도 진행할 수 있었는데
이력서 멘토링을 준비하기 위해 경력 정리한다고
진행하지 못한게 아쉽습니다.

## 🪜 다음주에 다르게 하고 싶은 것이 있다면?(액션 플랜과 함께)
이력서 멘토링을 신청하기 위해 경력 정리를 마무리하고, 다시 목표에 집중하고 싶네요.

**Comment by imakerjun** [comment]
# 📅 4월 12~19일의 회고
## 💡 인사이트 있었던 일
- 애자일 코칭 교육을 받으면서, 2가지 선택 사항 중 한 가지를 더 잘 선택하는 방법을 배웠다. 2가지 선택사항에 대한 각각의 가설을 두고 빠르게 검증할 수 있는 작은 단위를 고민해보고 실행해 보는 것!
- 학생과 함께 겪은 이야기를 5월에 책을 내는걸 목표로 잡으니 보다 더 함께 성장한다는 소속감, 그리고 글을 쓴다는 기쁨이 더 커졌다.
- 프론트엔드 스터디 리뷰 방식을 실시간으로 변경하면서 더 많은 커뮤니케이션과 피드백을 이끌어 낼 수 있게 되었다.

## 🤔 아쉬움이 남은 일
- 블로그 스터디 운영자인데, 블로그 글 업로드를 하지 못하였다... 이제부터는 화요일에 미리 업로드 하고 PR보내는 사이클을 가져야 겠다.
 
## 🪜 다음주에 다르게 하고 싶은 것이 있다면?(액션 플랜과 함께)
- 화요일 자바스크립트 스터디 후에 바로 글쓰기 PR 보내기

**Comment by younoah** [comment]
# 📅 4월 12~19일의 회고
## 📝 블로그 글 소개

리엑트에서 컴포넌트를 만드는 방법에 대해 정리했습니다.

## 💡 인사이트 있었던 일

리엑트를 공부하면서 아래와 같은 재미난 점을 알게 되었습니다!

- 리엑트는 컴포넌트이다.
- 리엑트의 컴포넌트는 상태가 변하면 새로 렌더링되어진다.
- 리엑트로 컴포넌트를 만드는 방법은 크게 클래스와 함수로 2가지 방식이 있다.
- pureComponent(클래스방식)과 memo(함수방식)을 이용하면 상태의 참조값이 변할때만 렌더링 되도록 할 수 있다.

## 🤔 아쉬움이 남은 일

너무 저만을 위한 글을 작성한 것 같습니다.
남들이 읽기 편하도록 좀 더 풀어 쓰고 글을 다듬어야겠다는 생각을 했습니다.

## 🪜 다음주에 다르게 하고 싶은 것이 있다면?(액션 플랜과 함께)
- 좀 더 남들이 읽기 편하도록 풀어써보기
- 그러기 위해서 우선 간단하게 정리하고 어린아이에게 개념을 설명한다는 생각으로 글을 작성해보기

**Comment by bigsaigon333** [comment]
# 📅 4월 12~19일의 회고
## 📝 블로그 글 소개
5번의 페어 프로그래밍을 하면서 회고를 매번 수행했지만, 그에 따른 액션 플랜은 한번도 도출하지 않았음을 깨달았습니다.
매번 마음속 한구석에는 아 저번 페어한테 이런 피드백 받았었지~ 주의해야지~ 하다가  막상 페어프로그래밍을 할때면 까마득히 잊어버리고 말았습니다. 매번 반복되는 피드백을 정리하여, 이제부터는 마음속에 새겨두려고 합니다. 

## 💡 인사이트 있었던 일
아이패드 에어4를 어제 사고, 오늘 배송받았습니다...! 백만원 넘는 돈을 투자하였으니 이를 십분 활용할 일만 남았습니다. 
이북도 많이 읽고, 필기도 많이 하고, 페어프로그래밍 시에도 적극 활용할 예정입니다!


## 🤔 아쉬움이 남은 일
오늘까지 페어 프로그래밍을 하다보니, 블로그 글을 완성을 못지었습니다. 액션 플랜을 떠올려보아야겠습니다.

## 🪜 다음주에 다르게 하고 싶은 것이 있다면?(액션 플랜과 함께)
블로그 주제를 일찍 선정하여서 가볍게 초안부터 적고 시작하고 싶습니다. 막바지에 되어서 허겁지겁 주제 찾고 글 벼락치기로 쓰고... 하는 패턴이 될려고 하는 조짐이 보이네요 ㅠ

**Reply by yujo11** [reply]
헐... 아이패드 대박!!👍👍👍

**Comment by joelonsw** [comment]
# 📅 4월 12~19일의 회고
## 📝 블로그 글 소개
SOLID를 처음 접하는 초심자의 시선에서 SOLID 원칙이 왜 등장했고, 왜 중요한지를 정리해봤습니다. 

## 💡 인사이트 있었던 일
SOLID에 대해 막상 설명하려니까 이게 왜 등장했지? 라는 질문에서 부터 막히더라고요. 
덕분에 객체지향의 장점에 대해 한 번 더 생각해볼 수 있었고, 자연스레 SOLID 원칙의 필요성도 깨닫게 되었던 것 같아요. 
Spring을 처음 배우고 있는데, SOLID에서 강조했던 사상(?)들이 조금씩 느껴지는 것 같아 Spring 학습에도 도움이 되는 포스팅이였던 것 같습니다.

## 🤔 아쉬움이 남은 일
사실 블로그 글을 하나 작성할 때마다 시간이 꽤 오래걸려요. 
글의 구조를 잡고, 무엇을 어찌 설명할지 생각하다보면 시간이 빨리가더라구요...
언젠간 조금 속도가 붙겠죠?

## 🪜 다음주에 다르게 하고 싶은 것이 있다면?(액션 플랜과 함께)
TIL을 깃헙에 작성하는데, 블로그 포스팅하고 싶은 주제가 있거나, 이걸 올리면 좋겠다 싶은 부분을 정리하면 좋을 것 같네요!

**Comment by catsbi** [comment]
# 📅 4월 12~19일의 회고
## 📝 블로그 글 소개
2021년 4월 3주차 회고록 작성입니다. 한 주의 회고록이지만, 블로그 스터디가 시작되기까지 있던 1~2주간 진행했던 사항들도 같이 정리되었으며, 부제는 다시 기본으로라는 타이틀로 신기술과 기능구현에만 치중해있다가 다시 기본으로 돌아가 처음부터 공부를 시작했습니다.

## 💡 인사이트 있었던 일
인프런에서 김영한님의 백엔드 로드맵을 따라가면서 자바와 스프링의 매력에 빠지게 되었는데, 
막연히 무겁고 둔하다고만 생각한 스프링/스프링부트가 어떻게 탄생했고 애노테이션하나하나까지 왜 개발되었고,
이러한 애노테이션이 없고스프링이 없을때는 어떻게 개발을 했고 객체지향의 원칙인 SOLID를 지켰는지 라이브 코딩을 통해
강의를 해주시는데, 너무 이해가 찰떡같이 되서 인사이트가 크게 되었습니다. 

## 🤔 아쉬움이 남은 일
내 집중력에 대한 한탄. 
회사가 바쁘지않아 회사에서도 죙일 인강보고 포스팅 집에서도 거의 인강보고 포스팅, 
주말에는 자바의정석 스터디를 진행하는데, 시간대비 효율이라고해야하나 결과물이 너무 적고, 계속 산만하게 다른짓을 하면서 시간을 죽이고있는데, 많이 아쉽습니다. 

## 🪜 다음주에 다르게 하고 싶은 것이 있다면?(액션 플랜과 함께)
다르게 하고싶으면서 또 다르게 하기싫은 부분인데, 바로 지금처럼 똑같이 공부를 하되, 산만하지 않게 집중하는 것입니다.

**Comment by lns13301** [comment]
# 📅 4월 12~19일의 회고
## 📝 블로그 글 소개
SOLID 에 대해서 간단하게 글을 작성 했습니다.

## 💡 인사이트 있었던 일
늘 정리하고자 생각은 했지만 실천하기가 쉽지 않았습니다.
블로그 스터디를 핑계삼아 미뤄뒀던 글을 작성 할 수 있었습니다.
다른 분들의 글을 읽으며 글을 표현하는 여러가지 방법을 알 수 있었습니다.

## 🤔 아쉬움이 남은 일
같은 주제를 준비한 조엘을 글을 보고나니 같은 주제인데 정말 표현하는 방식이 다를 수 있다는 것을 깨달았습니다.
나 혼자 본다고 생각하고 글을 작성하긴 했지만, 다른 사람도 읽는 다는 생각으로 좀 더 친절하게 글을 작성 해봐야겠습니다.

## 🪜 다음주에 다르게 하고 싶은 것이 있다면?(액션 플랜과 함께)
시간에 쫓기며 글을 작성 하지 말고 미리 글을 준비 해야겠습니다.

**Comment by kingjky** [comment]
# 📅 4월 12~19일의 회고
## 📝 블로그 글 소개
글 쓰는 것이 익숙하지 않아서, 첫 글이라는 핑계로 생각나는대로 간단한 일기를 써 보았습니다.

## 💡 인사이트 있었던 일
개발 과정이 어렵고 힘들 때마다 '그 동안 열심히 취업 준비한건 바로 이 고생 하려고 그랬던거다' 라는 생각을 하며 버틸 수 있었습니다.

## 🤔 아쉬움이 남은 일
업무가 끝나면 개인 공부를 해야겠다고 생각하지만, 뜻대로 되지 않는 것이 아쉽습니다. 요령이 생기면 더 편해질까요?

## 🪜 다음주에 다르게 하고 싶은 것이 있다면?(액션 플랜과 함께)
다른 분들의 글을 읽으니 대부분 기술 관련 정보를 담고 계신데 저도 다른 사람에게 지식을 전달할 수 있으면 좋겠다는 생각을 하며, 기술 포스트를 작성할 수 있도록 노력하겠습니다.

**Comment by yujo11** [comment]
# 📅 4월 12~19일의 회고
## 📝 블로그 글 소개
버퍼 기간을 가졌습니다.

## 💡 인사이트 있었던 일

- 페어 프로그래밍을 통한 인사이트
- 기존 공부 방식에 대한 성찰

## 🤔 아쉬움이 남은 일

- 할 일들의 우선순위를 정했는데 정한 순서대로 진행하지 않았다.
- 집중력이 많이 떨어져서 앉아있는 시간은 길었지만 투입 된 시간 대비 결과가 없었다.

## 🪜 다음주에 다르게 하고 싶은 것이 있다면?(액션 플랜과 함께)

- 집중력 높이기
    1. 할 일들의 우선순위를 확실히 정한다.
    2. 집중력 유지에 방해가 되는 요소를 제거한다.(핸드폰, 슬랙, 카톡, 메일)

**Comment by LimHanGyeol** [comment]
# 📅 4월 12~19일의 회고
## 📝 블로그 글 소개
버퍼 기간을 가졌습니다.

## 💡 인사이트 있었던 일
* 이력서 멘토링을 하며 공부를 하고 첫 포스팅 주제를 정했는데,
머리로는 이해하고 있어도 이걸 설명하는 것은 역시 다른 영역이라고 생각 들었습니다.
내가 설명하고 글을 작성하게 될 때 잘못된 정보를 알려주게 되는 건 아닐지 좀 더 경계심을 갖게 되었습니다.

## 🤔 아쉬움이 남은 일
* 이번 주 여러 일이 겹쳐 글 작성을 완료하지 못했습니다. 현재 하고 있는 것들을 빨리 정리하고 진행해야겠습니다 !

## 🪜 다음주에 다르게 하고 싶은 것이 있다면?(액션 플랜과 함께)
* 우선순위를 재정비하고 시간을 집약적으로 사용하기.

**Comment by kelly-choi** [comment]
# 📅 4월 12~19일의 회고
## 📝 블로그 글 소개
버퍼 기간을 가졌습니다.
## 💡 인사이트 있었던 일

- 5월에 있을 Adsp, Sqld 시험 준비 할 예정입니다
- Python을 사용하여 데이터를 수집하는 작업을 진행하고 있습니다
- 이번에 git을 처음 만들어서 사용법을 익히는 중입니다

## 🤔 아쉬움이 남은 일

- 내 머릿속에 있는 내용을 적는다는 것에 대한 부담감(?)이 아직 큰 것 같습니다. 이 부담감을 이겨내고 첫 글을 작성하는게 저에게는 큰 의미가 있을 것 같습니다.

## 🪜 다음주에 다르게 하고 싶은 것이 있다면?(액션 플랜과 함께)

- 시험까지 남은 기간 공부할 분량 정해보기
- git에 대한 기본 개념 공부하기
- 현재까지 내 경력과 기술 수준 파악해보기
